prototype_source/pt2e_quantizer.rst

```diff
@@ -141,7 +143,38 @@ parameters can be shared among some tensors explicitly. Two typical use cases ar
 
 ``SharedQuantizationSpec`` is designed for this use case to annotate tensors whose quantization
 parameters are shared with other tensors. Input of ``SharedQuantizationSpec`` is an ``EdgeOrNode`` object which 
-can be an input edge or an output value. 
+can be an input edge or an output value.
+
+.. note::
+   * Sharing is transitive
+
+     Some Tensors might be effectively using shared quantization spec due to (1) two nodes/edges are
+     configured to use SharedQuantizationSpec (2) there is existing sharing of some of the nodes
+
+     For example, let's say we have two ``conv`` nodes ``conv1`` and ``conv2``, and both of them are fed into a ``cat``
+     node. `cat([conv1_out, conv2_out], ...)` Let's say output of ``conv1``, ``conv2`` and the first input of ``cat`` are configured
+     with the same configurations of ``QuantizationSpec``, second input of ``cat`` is configured to use ``SharedQuantizationSpec``
+     with the first input.
+     
+     .. code-block::
+     
+       conv1_out: qspec1(dtype=torch.int8, ...)
+       conv2_out: qspec1(dtype=torch.int8, ...)
+       cat_input0: qspec1(dtype=torch.int8, ...)
+       cat_input1: SharedQuantizationSpec((conv1, cat))  # conv1 node is the first input of cat
+     
+     First of all, the output of ``conv1`` is implicitly sharing quantization parameter (and observer object)
+     with the first input of ``cat``, and same for output of ``conv2`` and the second input of ``cat``.
+     So since user configures the two inputs of ``cat`` to share quantization parameters, by transitivity,
+     ``conv2_out`` and ``conv1_out`` will also be sharing quantization parameters. In the observed graph, you
+     will see:
```

Comment:
```suggestion
     Therefore, since the user configures the two inputs of ``cat`` to share quantization parameters, by transitivity,
     ``conv2_out`` and ``conv1_out`` will also be sharing quantization parameters. In the observed graph, you
     will see the following:
     
```